src/runtime/contrib/cublas/cublas_json_runtime.cc

```diff
@@ -52,9 +54,7 @@ class CublasJSONRuntime : public JSONRuntimeBase {
   const char* type_key() const override { return "cublas_json"; }  // May be overridden
 
   void Run() override {
-    // TODO(masahi): Reuse the same handle across different subgraphs
-    cublasLtHandle_t handle;
-    cublasLtCreate(&handle);
+    CuBlasLtThreadEntry* entry_ptr = CuBlasLtThreadEntry::ThreadLocal();
```

Comment:
I removed `using` and replaced this line with

```
auto entry_ptr = tvm::contrib::CuBlasLtThreadEntry::ThreadLocal();
```

to workaround the compile error.